code-challenges.test.js

```diff
@@ -17,35 +17,166 @@
 
 const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// const colors2 = [
+//   "chartreuse",
+//   "indigo",
+//   "periwinkle",
+//   "ochre",
+//   "aquamarine",
+//   "saffron"
+// ]
+// // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+// describe("newColors", () => {
+//   it("create a function that takes an array and removes the first item and shuffles the remainder", () => {
+//     expect(newColors(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+//     expect(newColors(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+//   })
+// })
+
+//RED- Good failure
+// FAIL  ./code-challenges.test.js
+// newColors
+//   ✕ create a function that takes an array and removes the first item and shuffles the remainder
+
+// ● newColors › create a function that takes an array and removes the first item and shuffles the remainder
+
+//   ReferenceError: newColors is not defined
+
+//     30 | describe("newColors", () => {
+//     31 |   it("create a function that takes an array and removes the first item and shuffles the remainder", () => {
+//   > 32 |     expect(newColors(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+//        |     ^
+//     33 |     expect(newColors(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
```

Comment:
Make sure to check out the resources provided in the prompt to see what you can use for this test. The outcome will not be in the same order every time because we should be shuffling the array, so we cannot make a test that only tests for one specific outcome.